packages/create-catalyst/.eslintrc.cjs

```diff
@@ -8,6 +8,7 @@ const config = {
     'no-console': 'off',
     'import/no-named-as-default': 'off',
     '@typescript-eslint/naming-convention': 'off',
+    '@typescript-eslint/no-unused-vars': ['error', { caughtErrorsIgnorePattern: '^_' }],
```

Comment:
This will require a changeset to release a new version of this config.

Reply:
Was able to revert with the eslint tip above 👍